src/Processor.ts

```diff
@@ -244,9 +244,9 @@ export abstract class Processor {
       if ('align' in statement) {
         reset = true
         if (statement['align'] == 'center') {
-          this.printer.alignment = Align.Center
+          this.printer.setAlignment(Align.Center)
```

Comment:
ensure async await at every promise call or function (includes any call to printer)